api/src/main/java/org/pmiops/workbench/captcha/CaptchaVerificationServiceImpl.java

```diff
@@ -36,6 +41,21 @@ public CaptchaVerificationServiceImpl(
     this.captchaApiProvider = captchaApiProvider;
   }
 
+  @VisibleForTesting
+  public void mockLoginUrl(String loginUrl) {
```

Comment:
This is not ideal, should just remove these and use the same approach that is used in this test case (define a static WorkbenchConfig, then provide a prorotype scoped bean of it): https://github.com/all-of-us/workbench/blob/61036972900bae172325d6128b604d647eb05745/api/src/test/java/org/pmiops/workbench/api/SumoLogicControllerTest.java#L42

In the individual test cases, override the values directly on the static config with the config you want.